docs/static/Intro.md

```diff
@@ -1,7 +1,7 @@
 # Intro
 Nostrum is a an Elixir library that can be used to interact with Discord.
 
-Nostrum currently supports the latest stable version of Elixir, v. 1.7.
+Nostrum currently supports the latest stable version of Elixir, v. 1.9.
```

Comment:
Isn't latest stable 1.10?

Reply:
I just changed it to what mix.exs is set to

Reply:
Going to change the wording as well as some other fixes, stay tuned, very spicy commits inbound